x/blob/README.md

```diff
@@ -44,50 +44,9 @@ MaxSquareSize | uint32 | 128
 celestia-app tx blob payForBlob <hex encoded namespace> <hex encoded data> [flags]
 ```
 
-### Programmatic Usage
-
-There are tools to programmatically create, sign, and broadcast `MsgWirePayForBlob`s
-
-```go
-// create the raw WirePayForBlob transaction
-wpfbMsg, err := apptypes.NewWirePayForBlob(block.Header.NamespaceId, blob, 16, 32, 64, 128)
-if err != nil {
-    return err
-}
-
-// we need to create a signature for each `MsgPayForBlob`s that
-// could be generated by the block producer
-// to do this, we create a custom `KeyringSigner` to sign messages programmatically
-// which uses the standard cosmos-sdk `Keyring` to sign each `MsgPayForBlob`
-keyringSigner, err := NewKeyringSigner(keyring, "keyring account name", "chain-id-1")
-if err != nil {
-    return err
-}
-
-// query for account information necessary to sign a valid tx
-err = keyringSigner.QueryAccount(ctx, grpcClientConn)
-if err != nil {
-    return err
-}
-
-// generate the signatures for each `MsgPayForBlob` using the `KeyringSigner`,
-// then set the gas limit for the tx
-gasLimOption := types.SetGasLimit(200000)
-err = pfbMsg.SignShareCommitments(keyringSigner, gasLimOption)
-if err != nil {
-    return err
-}
-
-// Build and sign the final `WirePayForBlob` tx that now contains the signatures
-// for potential `MsgPayForBlob`s
-signedTx, err := keyringSigner.BuildSignedTx(
-    gasLimOption(signer.NewTxBuilder()),
-    wpfbMsg,
-)
-if err != nil {
-    return err
-}
-```
+For submitting PFB transaction via a light client's rpc, see [celestia-node's documention](https://docs.celestia.org/developers/node-api/#post-submit_pfd).
+
+While not directly supported, the steps in the [`SubmitPayForBlob`](https://github.com/celestiaorg/celestia-app/blob/a82110a281bf9ee95a9bf9f0492e5d091371ff0b/x/blob/payforblob.go) function can be reverse engineered to submit blobs programmatically.
```

Comment:
Is this up to date? It takes me to the removed `MsgWirePayForBlob`

Reply:
this is not update to date, but I think its something that we will have to handle in #1113 after we merge this, as the most up to date reference will be dead after this branch gets deleted

https://github.com/celestiaorg/celestia-app/issues/1113#issuecomment-1347726776